hazelcast/invocation.py

```diff
@@ -122,21 +122,21 @@ def shutdown(self):
     def _invoke_on_partition_owner(self, invocation, partition_id):
         owner_uuid = self._partition_service.get_partition_owner(partition_id)
         if not owner_uuid:
-            self.logger.debug("Partition owner is not assigned yet", extra=self._logger_extras)
+            _logger.debug("Partition owner is not assigned yet")
```

Comment:
Don't we need `if _logger.isEnabledFor(logging.DEBUG)` check here and around other debug messages in this script?

Reply:
We don't have this check in most places. It is necessary only if we pass an argument that results in some computation. For example, for
```python
_logger.debug("xyz %s", costly_call())
```
the check makes sense. But for
```python
_logger.debug("xyz %s", some_reference)
```
it is not necessary because the logging package is smart enough to do string interpolation lazily only if the logger is enabled for that log level.

But thanks to your comment, I saw that we were using lazy interpolation only on some places, there were lots of non-lazy interpolations of the form
```python
_logger.debug("xyz %s" % some_reference)
```
I changed them with the correct approach on https://github.com/hazelcast/hazelcast-python-client/pull/229/commits/238e963318db7f6ae74592294c47ac68d7c46cd5